src/ui/common/src/index.tsx

```diff
@@ -92,6 +92,9 @@ import {
   aqueductApi,
   useDagGetQuery,
   useDagResultGetQuery,
+  useNodeArtifactGetQuery,
```

Comment:
It's fine to add these here for now, but just an FYI, if these functions aren't used outside of the common module, they don't need to really go in here.

Really the only time you should put something here for export is if you wish to access from the app module.

This will change of course once we start working on enterprise, but we're not quite there yet.

Reply:
oh okay it's actually nice as we reduce the work needed to add a new route :)